### DISCUSSION TITLE
No publishable resources for tag [views]

### DISCUSSION THREAD

**Discussion by oishiiLatte** [spatie/laravel-medialibrary#3613] [Q&A]
I want to render responsive images with my own HTML. I tried to publish the views with this command:

`php artisan vendor:publish --provider="Spatie\MediaLibrary\MediaLibraryServiceProvider" --tag="views"`.

But the result is `No publishable resources for tag [views]`.

I'm in Laravel 11 now

**Comment by netzknecht** [comment]
Try `php artisan vendor:publish --provider="Spatie\MediaLibrary\MediaLibraryServiceProvider" --tag="media-library-views"`.
